.github/workflows/ruby_tests.yml

```diff
@@ -123,6 +123,10 @@ jobs:
         if: github.event_name != 'push'
         run: |
           bundle exec rake test:foreman_puppet
+      - name: Run access permission test
+        if: ${{ startsWith(matrix.foreman-core-branch, '3') && github.event_name != 'push' }}
+        run: |
+          bundle exec rake test TEST="test/unit/foreman/access_permissions_test.rb"
```

Comment:
btw we don't need to test all the routes, we can just test the engine routes like:
https://github.com/theforeman/foreman_statistics/blob/master/test/unit/foreman_statistics/access_permissions_test.rb
OR
https://github.com/theforeman/foreman_bootdisk/blob/master/test/unit/access_permissions_test.rb

Reply:
There already is https://github.com/theforeman/foreman_puppet/blob/master/test/unit/foreman_puppet/access_permissions_test.rb

Reply:
Shall we close the PR if the test is already there?

Reply:
this effort shown that the test doesn't test the integration with core routes.

So however we decide to rewrite core API routes (eighter directly in core, as I have that in my PR or with your PR here) that would be untested. So this test still brings value in testing that override.

Reply:
Right. Shall we move ahead and merge it, then?